### ISSUE TITLE
Flameshot closes when i try to open screenshot in side program

### ISSUE THREAD

**Issue by timsofteng** [flameshot-org/flameshot#694]
**Flameshot version**
flameshot 0.6.0 + git 20191001-2(Debian)
Compiled with Qt 5.12.5

**Describe the bug**
Bug appear when option when "Close after capture" option chosen. When I try to open screenshot in side aplication Flameshot force closed so I cannot chose app to sent my screenshot to work with it.

**System Information**
Debian 10 buster, stable
But Flameshot downloaded from testing branch

**Comment by describe19** [comment]
Yes, I was about to post an issue about this behaviour, I think this issue title should be renamed:
"Close after capture" closes flameshot before "choose another app to open the capture" and "upload the selection to imgur" complete.

**Comment by borgmanJeremy** [comment]
Confirmed on master.
